corehq/apps/app_manager/templates/app_manager/summary_new.html

```diff
@@ -0,0 +1,63 @@
+{% extends "style/base_section.html" %}
+{% load xforms_extras %}
+{% load url_extras %}
+{% load hq_shared_tags %}
+{% load i18n %}
+{% load compress %}
+{% load djangular_tags %}
+
+{% block js %}
+    <script type="text/javascript" src="{% new_static 'app_manager/js/app_summary.ng.js' %}"></script>
+{% endblock js %}
+
+{% block js-inline %} {{ block.super }}
```

Comment:
That seems fine to me. I don't actually know what the value behind marking something `inline` in the first place is other than organization (which doesn't seem like a good reason to make things complicated).